tests/filewriter_test.cpp

```diff
@@ -71,14 +71,14 @@ TEST_CASE("Writing a summary to file", "[FileWriter]")
     struct stat results;
     mlboard::FileWriter f1("_temp1_");
```

Comment:
Looks like missed the indentation from line 64 to 72. Please reindent it properly